lib/domain/SyntaxAggregator.js

```diff
@@ -132,6 +143,22 @@ export default class SyntaxAggregator extends SyntaxModel {
         }
     }
 
+    onRequestError(error) {
+        if (this.filterView) {
+            if (this.currentMode === LanguageFilterMode.NONE) {
```

Comment:
For requests that call this handler on error, test for:
- if just entering syntax filter initially on error, expect that items have been cleared
- if 'ENOTFOUND', expect that No internet connection reported
- For all other errors, expect that it shows error connecting to SyntaxDB